### PR TITLE
Update examples for targets in docs

### PR DESCRIPTION
**Pull request by bndabbs**
## Proposed Changes

This change updates the wording in the targets examples to improve clarity. It also fixes the file extension in the **applications** example.

### CODE REVIEW

File: docs/inventory.md [hunk 1]
```diff
@@ -50,11 +50,11 @@ A target is a file that lives under the [`inventory/targets`](#targets) subdirec
 
 What you do with a [**target**](#targets) is largely up to you and your setup. Common examples:
 
-* **clusters**: Each target to map to a cluster you have, and used to caputure all configurations needed for a given cluster. For instance `targets/clusters/production-cluster1.yml`
-* **applications**: When using **Kapitan** to manage **Kubernetes** applications, is to use a [**target**](#targets) file to define everything that you would normally deploy in a single namespace, including all its resources, scripts, secrets and documentation. For instance `targets/mysql.yaml`
+* **clusters**: Map each [**target**](#targets) to a cluster, caputuring all configurations needed for a given cluster. For instance: `targets/clusters/production-cluster1.yml`
```
Comment (ramaro) [inline docs/inventory.md:53]: typo in cap*u*turing

**Comment by ramaro** [comment]
Thanks @bndabbs just added one comment but other than that it looks good 👍🏼

File: docs/inventory.md (hunk 1 above)
Comment (bndabbs) [reply docs/inventory.md]: Good catch! Should be fixed now

**Comment by ramaro** [comment]
Thanks @bndabbs !

**Pull request merged**